src/routes/safe/components/AddressBook/index.jsx

```diff
@@ -73,13 +113,26 @@ const AddressBookTable = ({
             <TableRow
               tabIndex={-1}
               key={index}
-              className={cn(classes.hide, index >= 3 && index === sortedData.size - 1 && classes.noBorderBottom)}
+              className={cn(
+                classes.hide,
+                index >= 3
```

Comment:
I would extract this to a boolean variable with meaningful name. I understand why this is needed but may be hard for other developers.

Reply:
Done